### PR TITLE
[libass] Update to 0.15.1

### PR DESCRIPTION
**Pull request by Masaiki**
**Describe the pull request**

- #### What does your PR fix?  
  Update libass to 0.15.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All triplets are supported. Just like previous version.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/libass/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "libass",
-  "version-string": "0.15.0",
-  "port-version": 1,
+  "version-string": "0.15.1",
```
Comment (JonLiu1993) [inline ports/libass/vcpkg.json:3]: @Masaiki ,Please change "version-string" to "version".

File: versions/l-/libass.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "a56d7a46c85680df7ee4344b4f16e44fc5d6a725",
+      "version-string": "0.15.1",
```
Comment (JonLiu1993) [inline versions/l-/libass.json:5]: Please change "version-string" to "version".

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/libass/vcpkg.json (hunk 1 above)
Comment (Masaiki) [reply ports/libass/vcpkg.json]: Done.

File: versions/l-/libass.json (hunk 2 above)
Comment (Masaiki) [reply versions/l-/libass.json]: Done.

**Comment by Masaiki** [comment]
sorry for clicking the button by mistake.

**Comment by JonLiu1993** [comment]
> sorry for clicking the button by mistake.

It's ok,thanks for your contribution

**Comment by strega-nil-ms** [comment]
Cool, thanks @Masaiki :)

**Pull request merged**